Kamikazi-Deborah/group-one-web-project
# Ft/imprint

#### What does this PR do?

- Implements the imprint page

#### Description of Task to be completed?

- User can view the imprint page

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ft/imprint`

- Run `npm install` to install modules

- Run `npm run dev` to start the development server

- In your browser navigate to `/imprint.html` or click on the `Impressum` link in the footer to see the imprint page

#### What are the relevant Trello stories?

- link to the implemented [Trello](https://trello.com/c/oV0nT1Xv/13-imprint-page) story

#### Screenshots (if appropriate)
![image](https://github.com/Kamikazi-Deborah/group-one-web-project/assets/110498537/878fcccc-c1ce-44ac-a9c0-ccc247ed6fc3)
